src/inet/IPEndPointBasis.cpp

```diff
@@ -683,7 +683,7 @@ System::Error IPEndPointBasis::PostPacketBufferEvent(chip::System::Layer & aLaye
     if (error != INET_NO_ERROR)
     {
         // If PostEvent() failed, it has not taken ownership of the buffer, so we need to retake it so that it will be freed.
-        aBuffer.Adopt(buf);
+        (void) System::PacketBufferHandle::Adopt(buf);
```

Comment:
I am currently expecting that all `Get_ForNow()` calls will go away (TLV is the holdout); the only remaining code that needs that semantics is the LWIP interface, which has its own (conditionally compiled) `GetLwIPpbuf()`. `Release_ForNow()` is a bit harder to get rid of, because currently six platform BLE implementations use it. So, for this code I used `Release` so that `Get` can be removed sooner.

Since this case is otherwise unique, perhaps in the future it might work to have something like a `SystemPacketBuffer::HandOver()` passed a lambda, but I haven't thought that through yet.

Reply:
OK.  The "create an extra handle" thing is a bit annoying codesize-wise, but I am hoping that in the future we will do a better API here and fix that.

Reply:
I hadn't noticed until now, but this code is inside CHIP_SYSTEM_CONFIG_USE_LWIP, so it can use the existing `GetLwIPpbuf()`. I'll do that during the final `_ForNow` purge.